test/nistrng/rng.c

```diff
@@ -49,7 +49,7 @@ static void nistkat_update(const unsigned char *provided_data,
 {
   int i;
   int len = AES256_KEYBYTES + AES_BLOCKBYTES;
-  uint8_t tmp[len];
```

Comment:
Oof, I didn't realize we did this -- we never should have. Variable-length stack arrays are dangerous.

Reply:
note that this is code written by NIST.